libshaderc_spvc/CMakeLists.txt

```diff
@@ -94,11 +94,14 @@ shaderc_add_tests(
   TEST_PREFIX shaderc
   LINK_LIBS shaderc_spvc ${SPVC_LIBS}
   INCLUDE_DIRS include ${shaderc_SOURCE_DIR}/libshaderc/include ${SPIRV-Cross_SOURCE_DIR}/..
+    ${SPIRV-Cross_SOURCE_DIR}/../effcee
```

Comment:
So far you aren't using these. I would avoid adding them until a test actually needs them.  (We went almost 3 years in SPIRV-Tools without adding this.)

Reply:
I'm told one of the fixtures headers uses effcee.  So this actually is needed.